packages/server/src/sdk/app/rows/tests/utils.spec.ts

```diff
@@ -0,0 +1,90 @@
+import { generator } from "@budibase/backend-core/tests"
+import { FieldType, Table } from "@budibase/types"
+jest.mock("../../../../sdk", () => ({
+  views: {
+    ...jest.requireActual("../../../../sdk/app/views"),
+    get: jest.fn(),
+  },
+}))
+
+import sdk from "../../../../sdk"
+import { trimViewFields } from "../utils"
+
+const mockGetView = sdk.views.get as jest.MockedFunction<typeof sdk.views.get>
+
+describe("utils", () => {
+  const table: Table = {
+    name: generator.word(),
+    type: "table",
+    schema: {
+      name: {
+        name: "name",
+        type: FieldType.STRING,
+      },
+      age: {
+        name: "age",
+        type: FieldType.NUMBER,
+      },
+      address: {
+        name: "address",
+        type: FieldType.STRING,
+      },
+    },
+  }
+
+  beforeEach(() => {
+    jest.resetAllMocks()
+  })
+
+  describe("trimViewFields", () => {
```

Comment:
A test case which confirms the constant internal/external columns are maintained correctly would be good, as well as making sure that the `_viewId` is always removed as expected.

Reply:
This should not be the job of this trimViewFields. This is a more db logic. It's tested on the patch itself